test/components/rook/rook-ceph_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build packet_fluo
+// +build e2e
+
+package rook
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	testutil "github.com/kinvolk/lokomotive/test/components/util"
+)
+
+const timeout = time.Minute * 8
+
+func TestRookCephDeployment(t *testing.T) {
```

Comment:
It seems this test and all sub-tests should run in parallel.

Reply:
The sub tests are running in parallel see the t.Run func.

Reply:
But both can be set to parallel. There is a new linter for it :)